app/policies/roles/institution/health_organization.json.erb

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "name": "Institution <%= institution_name %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "institution/<%= institution_id %>",
+          "action": [
+            "institution:read",
+            "institution:update",
+            "institution:delete",
+            "institution:registerDevice",
+            "institution:createSite",
+            "institution:createEncounter",
+            "institution:createRole"
+          ],
+          "delegable": true
+        },
+        {
+          "resource": [
+            "site?institution=<%= institution_id %>",
+            "device?institution=<%= institution_id %>",
+            "testResult?institution=<%= institution_id %>",
+            "encounter?institution=<%= institution_id %>"
+          ],
+          "action": "*",
+          "delegable": true
+        },
+        {
+          "resource": "role?institution=<%= institution_id %>",
+          "action": "*",
+          "delegable": true
+        }
+      ]
```

Comment:
A health organization admin should have similar permissions to a health organization owner:
```json
{
      "resource": "institution/<%= institution_id %>",
      "action": [
        "institution:read",
        "institution:update",
        "institution:delete",
        "institution:createRole"
      ],
      "delegable": true
    },
    {
      "resource": "testResult?institution=<%= institution_id %>",
      "action": "testResult:medicalDashboard",
      "delegable": true
    },
    {
      "resource": "role?institution=<%= institution_id %>",
      "action": "*",
      "delegable": true
    }
```

Same applies to admins from all the other institution kinds. We should copy the permissions already defined for the owners.